doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -798,6 +798,10 @@ Bluetooth libraries and services
   * Updated:
 
     * Improved the :ref:`bt_fast_pair_readme` library documentation to include the description of the missing Kconfig options.
+    * Strengthen requirements regarding thread context of Fast Pair API calls.
+      More functions are required to be called in the cooperative thread context.
+    * Disallowed calling the :c:func:`bt_fast_pair_info_cb_register` function when the Fast Pair is enabled.
```

Comment:
```suggestion
    * The :c:func:`bt_fast_pair_info_cb_register` function is now not called when the Fast Pair is enabled.
```

Reply:
Maybe `is now not allowed to be called when the Fast Pair is enabled.`? That would be more accurate.